CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 # Contribution Guidelines
 
-Contributions to this repo are very welcome! We follow a fairly standard [pull request 
+Contributions to this repo are very welcome! We follow a fairly standard [pull request
```

Comment:
NIT: were there any changes to this file or just whitespace? If the latter, could you revert, as this PR is already big enough!

Reply:
A new one section was added, but the rest are whitespaces. I cleaned commits, and ignore those whitespaces in new commits.